config/backend-config/workspace-config.go

```diff
@@ -6,9 +6,11 @@ import (
 	"io/ioutil"
 	"net/http"
 
-	"github.com/rudderlabs/rudder-server/utils/logger"
+	"github.com/rudderlabs/rudder-server/utils/sysUtils"
 )
 
+var IoUtil sysUtils.IoUtilI = sysUtils.NewIoUtil()
```

Comment:
There is already an Ioutil (with lowercase u) in backend-config.go Can you use that instead?